src/ploigos_step_runner/step_implementers/push_container_image/skopeo.py

```diff
@@ -122,6 +122,7 @@ def _run_step(self):
         organization = self.get_value('organization')
         image_tar_file = self.get_value('image-tar-file')
         destination_url = self.get_value('destination-url')
+        dest_tls_verify = self.get_value('dest-tls-verify')
```

Comment:
Can we change this to `tls-verify` as well? As is, this won't inherit if specified in global config